src/ex4/server.js

```diff
@@ -1 +1,22 @@
-// Express boilerplate, hosting the `dist` file, connecting to the routes
\ No newline at end of file
+const express = require('express');
+var bodyParser = require('body-parser');
+const dotenv = require('dotenv');
+dotenv.config();
+
+const todoRouter = require('./server/routes/todo_router');
+const port = parseInt(process.env.PORT); //PORT FROM .ENV
```

Comment:
It is a best practice to add a default value in case there was a problem with reading the .env file.

`const port = parseInt(process.env.PORT) || 3000;`